{
  "name": "trycatch",
  "version": "1.5.20",
  "description": "An asynchronous domain-based exception handler with long stack traces for node.js",
  "homepage": "http://github.com/CrabDude/trycatch",
  "repository": {
    "type": "git",
    "url": "git://github.com/CrabDude/trycatch.git"
  },
  "bugs": {
    "url": "http://github.com/CrabDude/trycatch/issues"
  },
  "author": {
    "name": "Adam Crabtree",
    "email": "dude@noderiety.com",
    "url": "http://noderiety.com"
  },
  "main": "lib/trycatch",
  "directories": {
    "lib": "./lib/"
  },
  "engines": {
    "node": ">=0.8.14"
  },
  "engineStrict": true,
  "scripts": {
    "test": "npm run lint && npm run unittests",
    "lint": "jshint --config .jshintrc  `find . -path ./node_modules -prune -o -name '*.js' -print | tr '\n' ' '`",
    "unittests": "mocha  --reporter spec ./test/*.js && USE_STRICT=true mocha --reporter spec test/*"
  },
  "licenses": [
    {
      "type": "MIT",
      "url": "http://www.opensource.org/licenses/mit-license.html"
    }
  ],
  "dependencies": {
    "hookit": "^1.1.3",
    "ansi-styles": "1.x.x"
  },
  "optionalDependencies": {
    "ansi-styles": "1.x.x"
  },
  "keywords": [
    "error",
    "exception",
    "try",
    "catch",
    "stack",
    "trace"
  ],
  "devDependencies": {
    "jshint": "2.x.x",
    "lodash": "2.x.x",
    "mocha": "~1.4.0",
    "underscore": "~1.3.3",
    "use-strict": "1.x.x"
  },
  "gitHead": "fa5ca2b45f7ea6a8f23b0807d591ebeddc3532bf",
  "_id": "trycatch@1.5.20",
  "_shasum": "43e18944c2a57937dccc7adf541cb90af982c98f",
  "_from": "trycatch@latest",
  "_npmVersion": "2.14.2",
  "_nodeVersion": "4.0.0",
  "_npmUser": {
    "name": "crabdude",
    "email": "dude@noderiety.com"
  },
  "maintainers": [
    {
      "name": "crabdude",
      "email": "dude@noderiety.com"
    }
  ],
  "dist": {
    "shasum": "43e18944c2a57937dccc7adf541cb90af982c98f",
    "tarball": "http://registry.npmjs.org/trycatch/-/trycatch-1.5.20.tgz"
  },
  "_resolved": "https://registry.npmjs.org/trycatch/-/trycatch-1.5.20.tgz",
  "readme": "ERROR: No README data found!"
}
